src/HelixServer.js

```diff
@@ -240,6 +234,36 @@ class HelixServer extends EventEmitter {
       return;
     }
 
+    // check for helix query
+    rgx = HELIX_QUERY_REGEXP.exec(ctx.path);
+    if (rgx) {
+      const [, indexName, queryName] = rgx;
+      const { owner, repo } = ctx.strain.content;
+      const { algoliaAppID, algoliaAPIKey } = ctx.config;
+
+      const urlPath = ctx.config.indexConfig.getQueryURL(
```

Comment:
don't you want to catch the case when indexConfig is not defined? 
it could return a `404`?

Reply:
good point, I'll fix it.